README.md

```diff
@@ -247,25 +247,33 @@ Shows Deaths from all causes in comparison to the min, max and mean of Deaths fr
   python -m venv .venv
   .venv/bin/pip install -r requirements.txt
   ```
+- If you are using anaconda you will need to also run this to ensure the pip install works
+  ```sh
+  conda install -c conda-forge python-crfsuite
```

Comment:
prob suggest to run this _before_ `pip install -r requirements.txt`, otherwise it'll throw the build error?